qiskit_experiments/library/characterization/analysis/drag_analysis.py

```diff
@@ -180,14 +180,13 @@ def _evaluate_quality(self, fit_data: curve.FitData) -> Union[str, None]:
             - a DRAG parameter value within the first period of the lowest number of repetitions,
             - an error on the drag beta smaller than the beta.
         """
-        fit_beta = fit_data.fitval("beta").value
-        fit_beta_err = fit_data.fitval("beta").stderr
-        fit_freq0 = fit_data.fitval("freq0").value
+        fit_beta = fit_data.fitval("beta")
+        fit_freq0 = fit_data.fitval("freq0")
 
         criteria = [
             fit_data.reduced_chisq < 3,
-            fit_beta < 1 / fit_freq0,
-            fit_beta_err < abs(fit_beta),
+            fit_beta.nominal_value < 1 / fit_freq0.nominal_value,
+            fit_beta.std_dev < abs(fit_beta.nominal_value),
```

Comment:
Do we need to safeguard against nan here? Might be good to fix this while we are at it.

Reply:
Added utility since this appears everywhere https://github.com/Qiskit/qiskit-experiments/pull/551/commits/602bf7934418414e8cf0ec23b78ecec69815f41a